src/library/autodj/autodjprocessor.cpp

```diff
@@ -829,6 +829,7 @@ void AutoDJProcessor::playerPositionChanged(DeckAttributes* pAttributes,
 
 TrackPointer AutoDJProcessor::getNextTrackFromQueue() {
     // Get the track at the top of the playlist.
+    m_pAutoDJTableModel->select();
```

Comment:
The Shift+F12 issue after start-up is also solved if this is moved here: 
https://github.com/mixxxdj/mixxx/blob/1ca77e00a53bbd63e5bb899ac27b8897efa95200/src/library/autodj/autodjprocessor.cpp#L126

Reply:
And you need to add 
```
installEventFilter(m_pKeyboard);
```
here: 
https://github.com/mixxxdj/mixxx/blob/a74230c8c478e72e4d1af7ad0a3c0a2eb09388fe/src/mixxx.cpp#L164

Reply:
> ```
> mixxx.cpp#L164: installEventFilter(m_pKeyboard);
> ```

that did the trick! Thanks

Reply:
even though it's there, too https://github.com/mixxxdj/mixxx/blob/2.3/src/mixxx.cpp#L538